src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -89,6 +90,16 @@
    */
   private int requestSequenceId = 0;
 
+  /**
+   * Used to count number of writes that were skipped because the request was already cancelled
+   */
+  private int skippedWrites;
```

Comment:
Yes, I'll change it to volatile. I wanted to avoid that for performance reasons, but I guess most requests won't be cancelled.